clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -905,3 +911,20 @@ def _get_multipart_id(nb_parts: PhilipsNumberOfParts, run_num: str) -> Optional[
         raise ValueError(f"{run_num} is outside of the scope.")
     if nb_parts == PhilipsNumberOfParts.OTHER:
         return None
+
+
+def delete_real_and_imaginary_files(bids_folder: PathLike) -> None:
+    """Delete all files with a `_real` or `_imaginary` suffix from the BIDS folder.
+
```

Comment:
Could you add a little bit more context here ? 
For example : Where do these files come from ? Why are we removing them ? ...
I think you can use some of the sentences you wrote in the PR message.